### DISCUSSION TITLE
请问plugin模块里的文件怎么可以进行单独运行嘛

### DISCUSSION THREAD

**Discussion by LoveUSimone** [ctripcorp/flybirds#100] [Q&A] [answered]
请问，我想在本地使用plugins中的一些组件、API 和配置，应该怎么做呢

**Comment by xiaoxiaocaiiao** [comment]
你需要用 组件和api 实现什么样的功能？

**Reply by LoveUSimone** [reply]
就是想自己在本地写一个测试文件，然后使用这些功能，比如点击组件啊，ocr识别啥的

**Comment by xiaoxiaocaiiao** [comment, marked as answer]
API、组件需要配合框架一起使用，不可以脱离框架运行流程单独使用，除非使用里面的 一些公共方法 ，但是这样你需要理解调用参数并且自己赋值，不建议使用 。
